File: app216-zuul/src/CommandWords.java

/**
 * This contains all the possible command words
 * for the game of Zuul
 *
 * @author Derek Peacock & Nicholas Day
 * @version 25-08-2021
 */
public enum CommandWords
{
    //TODO add new command words
    MAP("map", "Display map!"),
    ITEMS ("item", ""),
    GO   ("go","Exit location in <direction>"),
    TAKE ("take", "Take <it+em> from location"),
    HELP ("help", "List all available commands"),
    QUIT ("quit", "End the game");


    public final String word;
    public final String description;
    
    CommandWords(String word, String description)
    {
        this.word = word;
        this.description = description;
    }
}

File: IJ-App06-Zuul/src/CommandWords.kt
import java.util.Locale
import java.util.HashMap
import kotlin.jvm.JvmStatic

/**
 * This contains all the possible command words
 * for the game of Zuul
 *
 * @author Derek Peacock & Nicholas Day
 * @version 25-08-2021
 */
enum class CommandWords(val word: String, val description: String) {
    GO("go", "Exit location in <direction>"), TAKE("take", "Take <item> from location"), HELP("help", "List all available commands"), QUIT("quit", "End the game");
}